core/authorship/impl/proposer_impl.cpp

```diff
@@ -60,10 +60,16 @@ namespace kagome::authorship {
       }
     }
 
+    auto remove_res = transaction_pool_->removeStale(parent_block_number);
+    if (remove_res.has_error()) {
+      SL_DEBUG(logger_,
```

Comment:
maybe change debug to error?

Reply:
Debug because that was non-critical, but you are right - it is still an error. Thus I will change it to SL_ERROR